src/OpenLoco/src/Ui/Window.h

```diff
@@ -208,8 +208,8 @@ namespace OpenLoco::Ui
         };
         uint16_t sortMode;
         uint16_t var_846 = 0;
-        uint16_t var_850 = 0;
-        uint16_t var_852 = 0;
+        uint16_t var_850 = 0; // vehicle list filter
+        uint16_t var_852 = 0; // vehicle list cargo; news ticks
```

Comment:
should we just rename these and remove the comment? same below

Reply:
I considered mucking about with unions for these, but @ZehMatt has some ideas for window-specific variable storage (without inheritance) that I think would be the better path forward.